components/esp_http_server/include/esp_http_server.h

```diff
@@ -49,7 +49,8 @@ initializer that should be kept in sync
         .global_transport_ctx_free_fn = NULL,           \
         .open_fn = NULL,                                \
         .close_fn = NULL,                               \
-        .uri_match_fn = NULL                            \
+        .uri_match_fn = NULL,                           \
+        .coreID = tskNO_AFFINITY                        \
```

Comment:
This syntax will work of course, but it will be nice if this is placed right below stack_size, just as in the httpd_config structure definition